docs/modules/cli/pages/integration-revealjs.adoc

```diff
@@ -0,0 +1,56 @@
+= Integrate with reveal.js
+
+In this section, you'll learn:
+
+* [x] How to install the required dependencies.
+* [x]  What attributes you must define.
+* [x] How to convert an AsciiDoc file to a _reveal.js_ presentation.
+
+== Prerequisites
+
+We assume that you already have a fully working version of Asciidoctor.js CLI installed and available in your `PATH`.
+If this is not the case, please read the xref:setup/install.adoc[installation instructions].
```

Comment:
I think you have an extra path segment in this xref. I don't see a "setup" folder.